package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "idempotency-redis",
-  "version": "1.2.3",
+  "version": "1.2.4",
```

This change should increase the minor version of the package.
```suggestion
  "version": "1.3.0",
```